network/p2p/p2pbuilder/libp2pscaler.go

```diff
@@ -16,14 +16,14 @@ func getNumFDs() (int, error) {
 	return int(l.Cur), nil
 }
 
-func allowedMemory(scaleFactor float64) (int64, error) {
+func AllowedMemory(scaleFactor float64) (int64, error) {
```

Comment:
```suggestion
func allowedMemory(scaleFactor float64) (int64, error) {
```

Tests can reference and run the unexported version of this function so why the need to export it?

Reply:
https://github.com/onflow/flow-go/pull/4846/commits/3bbaf2470a3825ea4e1ee72f7bc468c4870b3867